pkgs/top-level/python-packages.nix

```diff
@@ -7126,6 +7126,30 @@ in {
     };
   };
 
+  grammalecte = buildPythonPackage rec {
```

Comment:
Can you move this python package out of this file as described in the header of this file?